### PR TITLE
AC Test Fix - LRAC-14779 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://liferay.atlassian.net/browse/LRAC-14779
https://liferay.atlassian.net/browse/LRAC-13746
[Testray Result](https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2447452074)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by joshitagaki — APPROVED** [review]
Thanks Vini! LGTM! Just to clarify the macro, so it's changing the password to test1, but changes back to test if the variable is set?

**Comment by ViniLopes20** [comment]
> Thanks Vini! LGTM! Just to clarify the macro, so it's changing the password to test1, but changes back to test if the variable is set?

Yeah, if we pass the variable, then it will change the user's password to `test`. This is only necessary if it is for default AC users who cannot have their password changed to `test1` to avoid failing other tests and we only need to do this on the first time that the user accesses the admin page (the next accesses for the same test It will not be necessary)

**Review by nkkwon — APPROVED** [review]
LGTM! :+1:

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACAdmin.macro [hunk 1]
```diff
@@ -1,7 +1,20 @@
 definition {
 
 	@summary = "Default summary"
-	macro configureMailSettings(incomingPOPServer = null, userPassword = null, outgoingSMTPServer = null, userName = null, incomingPort = null, outgoingPort = null) {
+	macro changeACUserPassword(userScreenName = null,) {
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACAdmin.macro:4]: Hey Vini, isn't the userScreenName required for this function to work? If yes, I believe it shouldn't be null.

**Review by manuferreira — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACAdmin.macro (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACAdmin.macro:4]: Oh yeah, you're right. I think this was put by SF and I didn't remember to check. Thanks, Manu 🕵🏽‍♀️

Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACAdmin.macro]: Thank you!

**Review by manuferreira — APPROVED** [review]
LGTM! Thanks Vini :muscle:

**Comment by manuferreira** [comment]
ci:forward

**Pull request closed without merging**